### PR TITLE
PR[4:1] feat: integrate cobra with the --generate flag/command

### PR DESCRIPTION
**Pull request by Alex-itu**
This integrates the generate command, to generate an HTML file.
You use this by saying 'go run main.go generate '.
Resolves [#59](https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar/issues/59)

### CODE REVIEW

**Comment by Slug-Boi** [comment]
Same suggestion as on fetcher see:
https://github.com/Agile-Arch-Angels/decentralized-tech-radar_dev/pull/20#issuecomment-2041988965

**Review by August-Brandt — APPROVED** [review]
Look good

**Review by Slug-Boi — APPROVED** [review]
Looks good. Good to finally have cobra :)

File: docs/dev_docs/read_and_write.md
Comment (Haruyzal) [inline docs/dev_docs/read_and_write.md:1]: I'd like to suggest that we create `dev_docs` and `user_docs` template files for us devs, to ensure consistency in the future. Just a thonk :3

**Review by Haruyzal — APPROVED** [review]

File: docs/dev_docs/read_and_write.md
Comment (Haruyzal) [inline docs/dev_docs/read_and_write.md:1]: Think @Alex-itu and I talked about this needing to be updated :)

**Review by Haruyzal — CHANGES_REQUESTED** [review]

File: docs/dev_docs/read_and_write.md
Comment (Alex-itu) [reply docs/dev_docs/read_and_write.md:1]: Yep, and it has been updated.

**Comment by Alex-itu** [comment]
> Nice changes! I've got some suggestions :3

Approve both of does

**Review by Haruyzal — APPROVED** [review]

**Pull request merged**